client.go

```diff
@@ -544,6 +558,7 @@ type Version struct {
 	Revision string
 }
 
+// Version returns the containerd daemon's version that the client is connected to
```

Comment:
`returns the version of the containerd daemon that the client is connected to` ?